src/components/Temporary/ThemeSection.tsx

```diff
@@ -144,12 +153,21 @@ export const ThemeSection = () => {
         <div ref={containerRef}>
           <GridAlternating py={7} addCenterLine gridCellStyle={{ marginBottom: spacing }}>
             <m.div style={{ height: heightWrapper, marginBottom: shiftUp }} className="su-overflow-hidden">
-              <div className="su-w-fit su-mr-0 su-ml-auto">
-                <Heading as="h3" size="f6" font="druk" align="right" className="su-mb-01em">
+              <FlexBox direction="col" alignItems="center" className="su-w-fit su-mr-0 su-ml-auto">
+                <Heading as="h3" font="druk" align="right" className="su-mb-01em su-text-[8rem]">
                   Discovery
                 </Heading>
-                <m.div className="su-h-20 su-origin-top-right" style={{ scaleY: tabHeight, backgroundColor: limeChange }} />
-              </div>
+                <m.div
+                  className="su-w-fit su-border-t-[2.8rem] su-border-bottom-none su-overflow-hidden"
+                  style={{
+                    borderRightWidth: discoverBarWidthChange,
+                    borderLeftWidth: discoverBarWidthChange,
+                    borderTopColor: redToLime,
+                    borderLeftColor: transparentToLime,
+                    borderRightColor: transparentToLime,
+                  }}
+                />
```

Comment:
This is how I animate a triangle created by CSS borders to a bar

Reply:
Neat